chat-plugins/wifi.js

```diff
@@ -29,7 +28,7 @@ function clearGiveaway() {
 	if (wifiRoom) delete wifiRoom.plugin;
 }
 
-var QuestionGiveAway = (function () {
+var QuestionGiveAway = (function	() {
```

Comment:
A ***tab*** after `function`? Are you serious? They're supposed to be single spaces like they used to be!